gnucash/gnome/gnc-plugin-page-budget.c

```diff
@@ -859,19 +883,38 @@ estimate_budget_helper(GtkTreeModel *model, GtkTreePath *path,
 
     num_periods = gnc_budget_get_num_periods(priv->budget);
 
-    for (i = 0; i < num_periods; i++)
+    if (priv->useAvg && num_periods)
     {
-        num = recurrenceGetAccountPeriodValue(&priv->r, acct, i);
-        if (!gnc_numeric_check(num))
-        {
-            if (gnc_reverse_balance (acct))
-                num = gnc_numeric_neg (num);
+        num = xaccAccountGetBalanceChangeForPeriod(acct, 
+            recurrenceGetPeriodTime(&priv->r, 0, FALSE),
+            recurrenceGetPeriodTime(&priv->r, num_periods - 1, TRUE), TRUE);
+        num = gnc_numeric_div(num, 
+            gnc_numeric_create(num_periods, 1), GNC_DENOM_AUTO,
+            GNC_HOW_DENOM_SIGFIGS(priv->sigFigs) | GNC_HOW_RND_ROUND_HALF_UP);
```

Comment:
Formatting: Line up the second and third lines with the first argument.

Reply:
You still don't get the formatting spec.  This call should look like:
```C
       num = gnc_numeric_div(num, gnc_numeric_create(num_periods, 1),
                             GNC_DENOM_AUTO,
                             GNC_HOW_DENOM_SIGFIGS(priv->sigFigs) |
                             GNC_HOW_RND_ROUND_HALF_UP);
```
The non-complying alignment on the ```xaccAccountGetBalanceChangeForPeriod``` call is OK only because of the long ```recurrenceGetPeriodTime``` call would extend past column 80.